doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+### Column chart 
+
+The $PlatformShort$ Column Chart, or Column Graph is among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by vertical bars of equal width and differing lengths. They are ideal for showing variations in the value of an item over time, data distribution, sorted data ranking (high to low, worst to best). Data is represented using a collection of rectangles that extend from the top to bottom of the chart towards the values of data points. Learn more about our [column charts](chart-types-column.md).
+
+### Polar Area chart 
+
+The $PlatformShort$ Polar Area Chart or Polar Graph belongs to a group of polar charts and has a shape of a filled polygon which vertices or corners are located at the polar (angle/radius) coordinates of data points. The Polar Area Chart uses the same concepts of data plotting as the Scatter Chart but wraps data points around a circle rather than stretching them along a horizontal line. Like with other series types, multiple Polar Area Charts can be plotted in the same data chart and they can be overlaid on each other to show differences and similarities between data sets. Learn more about our [polar charts](chart-types-polar.md).
+
+### Scatter chart 
+
+The $PlatformShort$ Scatter Chart, or Scatter Graph, is used to show the relationship between two values using a Cartesian (X, Y) coordinate system to plot data. Each data point is rendered as the intersecting point of the data value on the X and Y Axis. Scatter charts draw attention to uneven intervals or clusters of data. They can highlight the deviation of collected data from predicted results and they are often used to plot scientific and statistical data. The $PlatformShort$ Scatter chart organizes and plots data chronologically (even if the data is not in chronological order before binding) on X-Axis and Y-Axis. Learn more about our [scatter charts](chart-types-scatter.md).
+
+### Spline Chart 
+
+The $PlatformShort$ Spline Chart, or Spline Graph is a type of category line graph shows the continuous data values represented by points connected by smooth line segments of one or more quantities over a period time for showing trends and performing comparative analysis. The Y-Axis (labels on left side) show a numeric value, while the X-Axis (bottom labels) are showing a time-series or comparison category. You can include one or more data sets to compare, which would render as multiple lines in the chart. The $PlatformShort$ Spline chart is identical to the $PlatformShort$ Spline chart, the only different being the line chart is points connected by straight lines, and the spline chart points are connected by smooth curves. Learn more about our [spline charts](chart-types-spline.md).
+
```

Comment:
In order to have somewhat different samples we can use this one - Stacked Spline chart:

https://share.goabstract.com/1daa2f11-ec03-4828-a4c3-a714626dffdf?mode=build&sha=715a65597e8a2124aa55159d657ecbcda35fd8db

The data is the same from the Stepline below.

Reply:
ok, we are creating this sample